src/vma/sock/sockinfo.cpp

```diff
@@ -196,6 +196,18 @@ int sockinfo::ioctl(unsigned long int __request, unsigned long int __arg) throw
 		}
 		break;
 
+	case FIONREAD:
+		{
+			si_logdbg("request=FIONREAD, arg=%d", *p_arg);
```

Comment:
I think that it could be on fast path in user`s application so I would prefer `func`